README.md

```diff
@@ -1778,6 +1778,35 @@ Cromwell also accepts two [workflow option](#workflow-options) related to call c
 
 > **Note:** If call caching is disabled, the to workflow options `read_from_cache` and `write_to_cache` will be ignored and the options will be treated as though they were 'false'.
 
+## Docker Tags
+
+Docker tags are a convenient way to point to a version of an image (ubuntu:14.04), or even the latest version (ubuntu:latest).
+For that purpose, tags are mutable, in a sense that the image they point to can change, while the tag name stays the same.
+While this is very convenient in some cases, using mutable, or "floating" tags in WDL affects the reproducibility of the WDL file: the same WDL using "ubuntu:latest" run now, and a year, or even a month from now will actually run with different docker images.
+This has an even bigger impact when Call Caching is turned on in Cromwell, and could lead to false positive cache hits.
```

Comment:
The phrasing "when Cromwell uses the incorrect file" makes it sound like this is a thing that happens, which it should never.  It *would* happen if we allowed floating tags to be used for caching, which is why we don't.